### DISCUSSION TITLE
Why QueryStoreAfter does not support tenant configuration override as QueryIngesterWithIn

### DISCUSSION THREAD

**Discussion by shichanglin5** [grafana/mimir#7730] [Help and support] [answered]
<img width="912" alt="image" src="https://github.com/grafana/mimir/assets/48707349/eb5153aa-984c-405a-a828-cae7bf027833">

**Comment by dimitarvdimitrov** [comment, marked as answer]
it hasn't been added yet. I think it's a change we can accept. There's a guide to converting global setting to a per-tenant setting https://github.com/grafana/mimir/blob/4897e22e4fff33a9bb52325f2e7440a7e8147ebf/docs/internal/contributing/how-to-convert-config-to-per-tenant-limit.md#L12-L11
